tests/php/Controllers/Root/RootController/TopActionTest.php

```diff
@@ -4,11 +4,11 @@
 use Illuminate\Http\Request;
 
 /**
- * RootControllerクラスのテストを行う.
+ * RootControllerクラスのtopActionのテストを行う.
  *
  * @SuppressWarnings(PHPMD.TooManyPublicMethods)
  */
-class RootControllerTest extends TestCase
+class TopActionTest extends TestCase
 {
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)